backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -145,6 +145,19 @@ data class CellInfo(
     }
 }
 
+/**
+ * Describes the label to be added to the cell
+ *
+ * @param text the text to be displayed
+ * @param icon the icon to be displayed
+ * @param iconTint the tint of the icon
+ */
+data class CellLabel(
```

Comment:
can we have a text and an icon at the same time? If not maybe a sealed class could make it easier to understand

Reply:
Following up from the previous comment, if the intention is to make it possible to have both a label and an icon, then we can just leave the text out of here and rename this to CellIcon to wrap around the icon id and the tint.